tests/cli/test_main_list.py

```diff
@@ -117,3 +127,69 @@ def test_list_explicit(tmp_env: TmpEnvFixture, conda_cli: CondaCLIFixture):
         stdout, _, _ = conda_cli("list", "--prefix", prefix, "--explicit", "--auth")
         assert curl["dist_name"] in stdout
         assert "/t/some-fake-token/" in stdout  # with --auth we do
+
+
+@pytest.mark.integration
+def test_export(
+    tmp_env: TmpEnvFixture,
+    conda_cli: CondaCLIFixture,
+    path_factory: PathFactoryFixture,
+    monkeypatch: MonkeyPatch,
+):
+    """Test that `conda list --export` output can be used to create a similar environment."""
+    monkeypatch.setenv("CONDA_CHANNELS", "defaults")
+    reset_context()
+    # assert context.channels == ("defaults",)
```

Comment:
Should this commented line be removed?

Reply:
replaced `monkeypatch` with `mocker` instead since `context.channels` is one of those appending fields